src/main/java/org/reaktivity/nukleus/oauth/internal/OAuthRealms.java

```diff
@@ -205,35 +231,81 @@ private OAuthRealm(
         {
             assert nextRealmBit < MAX_REALMS;
             this.realmName = realmName;
-            this.realmId = 1L << nextRealmBit++ << MAX_SCOPES;
         }
 
         private long resolve(
-            String[] scopeNames)
+            String[] scopeNames,
+            String issuerName,
+            String audienceName)
```

Comment:
Recommend refactoring the order of these parameters to `issuerName`, `audienceName`, `scopeNames`.